src/InputWithLabel.js

```diff
@@ -0,0 +1,29 @@
+import React, {useRef, useEffect} from 'react';
+
+const InputWithLabel = ({
+    id,
+    value,
+    type='text',
+    onInputChange,
+    children
+    }) => {
+
+    const inputRef = useRef();
+    useEffect(() => {
+        inputRef.current.focus();
+    }, []);
```

Comment:
Just one small change here: 

We want the effect to be applied every render, not just the initial render. For that to be the case, we need a fully `undefined` dependency array argument to this `useEffect`, vs the empty array you have currently. With the empty array, it's only triggering on the initial render
```suggestion
    });
```

Reply:
Thank you, Darga. Feedback well noted and changes made. Appreciate your guidance.